libcxx/include/__mutex/unique_lock.h

```diff
@@ -74,13 +74,7 @@ class _LIBCPP_TEMPLATE_VIS unique_lock {
   }
 
   _LIBCPP_HIDE_FROM_ABI unique_lock& operator=(unique_lock&& __u) _NOEXCEPT {
```

Comment:
Typically for this LWG issue you need to update the synopsis due to the adding of `noexcept`, but that seems already done.